app/common/helpers/collections.py

```diff
@@ -177,9 +177,9 @@ def all_visible_questions(self) -> dict[UUID, "Question"]:
     @property
     def status(self) -> SubmissionStatusEnum:
         awaiting_sign_off = any(
-            x.key == SubmissionEventKey.SUBMISSION_SENT_FOR_CERTIFICATION for x in self.ordered_events
+            x.event_type == SubmissionEventType.SUBMISSION_SENT_FOR_CERTIFICATION for x in self.ordered_events
```

Comment:
Don't we need to be getting the latest event here, not just see if any event exists, and checking the `data`? Or is that in the subsequent PR?